config/environments/staging.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
```

Comment:
This and `uat.rb` are very similar. We can probably make a production-like config file that is used by production.rb, staging.rb and uat.rb to reduce duplication.